bitbots_vision/src/bitbots_vision/vision_modules/obstacle.py

```diff
@@ -1,8 +1,9 @@
+import itertools
+import numpy as np
+import rospy
```

Comment:
If I remember our coding standards correctly those non python standard, and not written by us libraries should be last in the list

Reply:
Afaik it's

1. Standard lib
2. Lib
3. In package

Reply:
@timonegk also agrees with me on this ^^

Reply:
It's in [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) and our styleguide is compliant with PEP8 (except for the maximum line length).